fuel-core/src/service/modules.rs

```diff
@@ -1,6 +1,7 @@
 use crate::config::Config;
 use crate::database::Database;
 use anyhow::Result;
+use fuel_core_interfaces::relayer::RelayerDb;
```

Comment:
[nit] Unrelated to this PR, but what do you think about moving the RelayerDb trait to the relayer crate? Since fuel-core already depends on the relayer, it would largely just be changing the import paths. My thinking is that we don't expect other modules outside of the relayer to access the RelayerDb directly, they should only use the Sender interface. So we probably shouldn't share guarded / lower level implementation requirements of modules with each other.

Reply:
The best way would be to create `fuel-storage` and extract database related stuff from `fuel-core`. In this case leaving `RelayerDb` in `fuel-core-interface` makes sense. I feel like those traits for db are going to be refactored in future to aggregate them and make better intention/clarification for usage, but for now, they seem good enough.

Reply:
i meant `fuel-core-storage` or `fuel-database` as we already have `fuel-storage`